apps/meteor/app/file-upload/server/config/GoogleStorage.ts

```diff
@@ -10,7 +10,17 @@ import '../../ufs/GoogleStorage/server';
 
 const get: FileUploadClass['get'] = async function (this: FileUploadClass, file, req, res) {
 	const { query } = URL.parse(req.url || '', true);
-	const forceDownload = typeof query.download !== 'undefined';
+	let forceDownload = typeof query.download !== 'undefined';
```

Comment:
wdyt of making this logic a lib since we're using the exact same as AmazonS3? 👀

Reply:
Dont know how to do that and dont know if is necessary to just two use cases.

Reply:
I agree with @KevLehman, it doesn't need to be a lib, just a single function to make sure we have the logic in a single place. You can create a new file under `file-upload/server/config/helper.ts` or something like that.